pkg/catalog/catalog.go

```diff
@@ -917,7 +913,7 @@ func (c *Catalog) ListCommits(ctx context.Context, repository string, branch str
 func (c *Catalog) Revert(ctx context.Context, repository string, branch string, params RevertParams) error {
 	repositoryID := graveler.RepositoryID(repository)
 	branchID := graveler.BranchID(branch)
-	ref := graveler.Ref(params.Reference)
+	revertRef := graveler.Ref(params.Reference)
```

Comment:
confusing name, sounds like a function. why not let it be ref? (same for other functions)

Reply:
renamed to `reference`. `ref` conflicts with the package name.